protocol/v2/ssv/validator/task_unique_id.go

```diff
@@ -0,0 +1,14 @@
+package validator
+
+import (
+	"fmt"
+
+	spectypes "github.com/bloxapp/ssv-spec/types"
+
+	"github.com/bloxapp/ssv/protocol/v2/ssv/runner"
+)
+
+func getTaskUniqueID(runner runner.Runner, duty *spectypes.Duty) string {
```

Comment:
move to logging package and name it
```suggestion
func TaskID(runner runner.Runner, duty *spectypes.Duty) string {
```

so we use it like `logging.TaskID`, maybe even naming it `logging.DutyID` is better because task is very generic

Reply:
done